lib/css_cleaner.rb

```diff
@@ -104,7 +104,7 @@ def sanitize_css_property(property)
   #   empty property returned.
   def sanitize_css_declaration_value(property, value)
     clean = ""
-    property.downcase!
+    property = property.downcase
```

Comment:
Is this still relevant now that we're no longer updating the CSS parser gem?

Reply:
Evidently not since the tests all passed when I reverted the change 👍